1_The_Physical/Guess_Point_Level_1_Stateless/2_Arrange_Act_Assert/2_1_Password_Validator/src/index.spec.ts

```diff
@@ -52,4 +52,13 @@ describe("password validator", () => {
     expect(result.errors).toContain("UppercaseMissing");
     expect(result.errors).toContain("PasswordTooShort");
   });
+
+  it("should return 'password too short' and not 'uppercase missing' or 'missing digit' for passwords like 'Two2' that are less than 5 characters but do contain an uppercase character and a digit", () => {
+    const passwordToCheck = "Two2";
+    const result = validatePassword(passwordToCheck);
+    expect(result.valid).toBeFalsy();
+    expect(result.errors).toContain("PasswordTooShort");
+    expect(result.errors).not.toContain("DigitMissing");
```

Comment:
Instead of having to explicitly check that the errors do not contain an error, you could do something like the following. 

`
expect(result).toEqual({
  valid: false,
  errors: ['PasswordTooShort']
)}
`